lib/receipt.spec.js

```diff
@@ -228,20 +226,20 @@ describe('Receipt', () => {
             it('can not be de-serialized', () => {
                 expect(() => Receipt.from({}, wallet)).to.throw();
             });
+        });
 
-            context('a de-serialized incomplete Receipt', () => {
-                [
-                    payload => delete payload.sender.fees.single,
-                    payload => delete payload.sender.balances.current,
-                    payload => delete payload.nonce
-                ].forEach(modifier => {
-                    context('when ' + modifier.toString(), () => {
-                        it('can not be signed', () => {
-                            let modifiedPayload = {...unsignedPayload};
-                            modifier(modifiedPayload);
-                            let incompleteReceipt = Receipt.from(modifiedPayload, wallet);
-                            expect(incompleteReceipt.sign()).to.be.rejected;
-                        });
+        context('a de-serialized incomplete Receipt', () => {
+            [
+                payload => delete payload.sender.fees.single,
+                payload => delete payload.sender.balances.current,
+                payload => delete payload.nonce
+            ].forEach(modifier => {
+                context('when ' + modifier.toString(), () => {
+                    it('can not be signed', () => {
+                        let modifiedPayload = {...unsignedPayload};
```

Comment:
Keep in mind that this is only a shallow-copy. Would it be possible to use a JSON stringify+parse clone in order to make sure the tests and their outcomes are independent of each other?